core/templates/site/admin/emailSentPage.gohtml

```diff
@@ -1,5 +1,5 @@
 {{ template "head" $ }}
-[<a href="/admin">Admin:</a> <a href="/admin/email/sent">(This page/Refresh)</a> | <a href="/admin/email/queue">Queue</a> | <a href="/admin/email/failed">Failed</a>]<br />
+[<a href="/admin">Admin:</a> <a href="/admin/email/sent">(This page/Refresh)</a> | <a href="/admin/email/queue">Queue</a> | <a href="/admin/email/failed">Failed</a>]
```

Comment:
The `<br/>` was correct, something about the way `<a>` is being used that is causing the text to break over multiple lines.

Reply:
@jules add the br back that isn't the issue.